docs/basics.txt

```diff
@@ -1,10 +1,8 @@
 [[basics]]
-JanusGraph Basics
-=================
+= JanusGraph Basics
 
 [[configuration]]
-Configuration
--------------
+== Configuration
 
 A JanusGraph graph database cluster consists of one or multiple JanusGraph instances. To open a JanusGraph instance a configuration has to be provided which specifies how JanusGraph should be set up.
```

Comment:
We should start an issue about the use of the word database cluster as it causes confusion. JanusGraph is a layer of abstraction on top of existing persistence technologies. Except for embedded databases like in-memory, persistit, berkeleyje and tupl, the underlying database is usually separated from the JanusGraph process by a network interface, if not on different hosts (thinking about BigTable HBase #116 , [EMR clusters with HBase](http://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-hbase-create.html), [DynamoDB](https://github.com/awslabs/dynamodb-titan-storage-backend) and managed Cassandra clusters ([Rackspace](https://www.rackspace.com/data/managed-nosql/cassandra-hosting), [Instaclustr](https://www.instaclustr.com/managed/cassandra/). Maybe the more general phrase "A JanusGraph server cluster consists of one or more JanusGraph server instances." is accurate.